### PR TITLE
oskar/datadog_RUM

### PR DESCRIPTION
**Pull request by oskar-binary**
# Description

* load data dog RUM script in <head> for production

### CODE REVIEW

File: src/components/containers/seo.js [hunk 1]
```diff
@@ -146,7 +146,19 @@ const SEO = ({ description, meta, title, no_index }) => {
                       ]
                     : []),
             ].concat(meta)}
-        ></Helmet>
+        >
+            <script
```
Comment (mustofa-binary) [inline src/components/containers/seo.js]: so we will still load datadog js in localhost?

Comment (oskar-binary) [reply src/components/containers/seo.js]: @mustofa-binary loading the the script yes, but it should only run in production from the `window.location.hostname === 'www.deriv.com' &&...` check

Comment (mustofa-binary) [reply src/components/containers/seo.js]: is it possible to defer?

Comment (oskar-binary) [reply src/components/containers/seo.js]: I think we don't want to defer since data dog will measure load time start and end times

Comment (mustofa-binary) [reply src/components/containers/seo.js]: ah, i see, alright then 👍

**Pull request merged**